cmd/agent/api/listener.go

```diff
@@ -13,5 +13,5 @@ import (
 
 // getListener returns a listening connection
 func getListener() (net.Listener, error) {
-	return net.Listen("tcp", fmt.Sprintf("localhost:%v", config.Datadog.GetInt("cmd_port")))
+	return net.Listen("tcp", fmt.Sprintf("0.0.0.0:%v", config.Datadog.GetInt("cmd_port")))
```

Comment:
Are we OK with this change security-wise or should it be behind a config setting?

Reply:
looking into the config I found that we have a `cmd_host` parameter (default to `localhost`) but we're note using it to configure the server.

wdyt of 
```golang
// getListener returns a listening connection
func getListener() (net.Listener, error) {
	return net.Listen("tcp", fmt.Sprintf("%v:%v", config.Datadog.GetString("cmd_host"), config.Datadog.GetInt("cmd_port")))
}
```